### ISSUE TITLE
Getting missingMethodException in remoting setup after updating packages

### ISSUE THREAD

**Issue by kgday** [fsbolero/Bolero#263]
My client side was working fine until I  I updated the packages. now getting a missing method exception - Method not found: 'Void System.Text.Json.Serialization.JsonFSharpConverter..ctor ..

The output of paket with the update:
```
C:\Development\Projects\Upwork\WGD>dotnet paket update
Paket version 7.1.5+e2fe7c693356a5e70a74cc01501f7dc5f8996695
Resolving dependency graph...
Updated packages:
  Group: Main
    - Bolero: 0.20.17 (added)
    - Bolero.Build: 0.20.17 (added)
    - Bolero.HotReload: 0.20.4 (added)
    - Bolero.HotReload.Server: 0.20.4 (added)
    - Bolero.Server: 0.20.17 (added)
    - Dapper: 2.0.123 (added)
    - Dapper.FSharp: 3.2.0 (added)
    - Elmish: 3.1.0 (added)
    - Evolve: 3.0.0 (added)
    - FSharp.Core: 6.0.5 (added)
    - FSharp.SystemTextJson: 0.18.24 (added)
    - HtmlAgilityPack: 1.11.43 (added)
    - Microsoft.AspNetCore.Authorization: 6.0.6 (added)
    - Microsoft.AspNetCore.Components: 6.0.6 (added)
    - Microsoft.AspNetCore.Components.Analyzers: 6.0.6 (added)
    - Microsoft.AspNetCore.Components.Forms: 6.0.6 (added)
    - Microsoft.AspNetCore.Components.Web: 6.0.6 (added)
    - Microsoft.AspNetCore.Components.WebAssembly: 6.0.6 (added)
    - Microsoft.AspNetCore.Components.WebAssembly.DevServer: 6.0.6 (added)
    - Microsoft.AspNetCore.Components.WebAssembly.Server: 6.0.6 (added)
    - Microsoft.AspNetCore.Connections.Abstractions: 6.0.6 (added)
    - Microsoft.AspNetCore.Http.Connections.Client: 6.0.6 (added)
    - Microsoft.AspNetCore.Http.Connections.Common: 6.0.6 (added)
    - Microsoft.AspNetCore.Metadata: 6.0.6 (added)
    - Microsoft.AspNetCore.SignalR.Client: 6.0.6 (added)
    - Microsoft.AspNetCore.SignalR.Client.Core: 6.0.6 (added)
    - Microsoft.AspNetCore.SignalR.Common: 6.0.6 (added)
    - Microsoft.AspNetCore.SignalR.Protocols.Json: 6.0.6 (added)
    - Microsoft.Extensions.Configuration: 6.0.1 (added)
    - Microsoft.Extensions.Configuration.Abstractions: 6.0.0 (added)
    - Microsoft.Extensions.Configuration.Binder: 6.0.0 (added)
    - Microsoft.Extensions.Configuration.FileExtensions: 6.0.0 (added)
    - Microsoft.Extensions.Configuration.Json: 6.0.0 (added)
    - Microsoft.Extensions.DependencyInjection: 6.0.0 (added)
    - Microsoft.Extensions.DependencyInjection.Abstractions: 6.0.0 (added)
    - Microsoft.Extensions.Features: 6.0.6 (added)
    - Microsoft.Extensions.FileProviders.Abstractions: 6.0.0 (added)
    - Microsoft.Extensions.FileProviders.Physical: 6.0.0 (added)
    - Microsoft.Extensions.FileSystemGlobbing: 6.0.0 (added)
    - Microsoft.Extensions.Http: 6.0.0 (added)
    - Microsoft.Extensions.Logging: 6.0.0 (added)
    - Microsoft.Extensions.Logging.Abstractions: 6.0.1 (added)
    - Microsoft.Extensions.Options: 6.0.0 (added)
    - Microsoft.Extensions.Primitives: 6.0.0 (added)
    - Microsoft.JSInterop: 6.0.6 (added)
    - Microsoft.JSInterop.WebAssembly: 6.0.6 (added)
    - Npgsql: 6.0.4 (added)
    - System.Diagnostics.DiagnosticSource: 6.0.0 (added)
    - System.IO.Pipelines: 6.0.3 (added)
    - System.Runtime.CompilerServices.Unsafe: 6.0.0 (added)
    - System.Text.Encodings.Web: 6.0.0 (added)
    - System.Text.Json: 6.0.5 (added)
    - System.Threading.Channels: 6.0.0 (added)
Installing into projects:
Created dependency graph (53 packages in total)
Downloading Microsoft.AspNetCore.Http.Connections.Common 6.0.6
..etc..
```
The exception occurs in my startup configure method which likes like below:
```    member this.Configure(app: IApplicationBuilder, env: IWebHostEnvironment, migrator: DB.DbMigrator) =
        migrator.Migrate()

        app
            .UseAuthentication()
            .UseRemoting()
            .UseStaticFiles()
            .UseRouting()
            .UseBlazorFrameworkFiles()            
            .UseEndpoints(fun endpoints ->
#if DEBUG
                endpoints.UseHotReload()
#endif
                endpoints.MapControllers() |> ignore

                endpoints.MapFallbackToBolero(Index.page)
                |> ignore)
        |> ignore
```

The exception occurs on the app.UseAuthentication etc line (from the stack trace I suspect the .UserRemoting is the culprint)
The full exception message is:

Method not found: 'Void System.Text.Json.Serialization.JsonFSharpConverter..ctor(System.Text.Json.Serialization.JsonUnionEncoding, System.String, System.String, System.Text.Json.JsonNamingPolicy, Boolean, Boolean, Boolean, System.Collections.Generic.IDictionary`2<System.Type,System.Text.Json.Serialization.JsonFSharpOptions>)'.

The exception stack trace is:
```
 at Bolero.Remoting.Server.RemotingService..ctor(PathString basePath, Type ty, Object handler, FSharpOption`1 configureSerialization)
   at <StartupCode$Bolero-Server>.$Remoting.AddRemoting@202-9.Invoke(IServiceProvider services)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitFactory(FactoryCallSite factoryCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitRootCache(ServiceCallSite callSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteVisitor`2.VisitCallSite(ServiceCallSite callSite, TArgument argument)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitIEnumerable(IEnumerableCallSite enumerableCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitRootCache(ServiceCallSite callSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteVisitor`2.VisitCallSite(ServiceCallSite callSite, TArgument argument)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.Resolve(ServiceCallSite callSite, ServiceProviderEngineScope scope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.CreateServiceAccessor(Type serviceType)
   at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.GetService(Type serviceType, ServiceProviderEngineScope serviceProviderEngineScope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.GetService(Type serviceType)
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetRequiredService(IServiceProvider provider, Type serviceType)
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetRequiredService[T](IServiceProvider provider)
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetServices[T](IServiceProvider provider)
   at Bolero.Remoting.Server.ServerRemotingExtensions.UseRemoting(IApplicationBuilder this)
   at WGD.Server.Startup.Configure(IApplicationBuilder app, IWebHostEnvironment env, DbMigrator migrator) in C:\Development\Projects\Upwork\WGD\src\WGD.Server\Startup.fs:line 68
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Span`1& arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at Microsoft.AspNetCore.Hosting.ConfigureBuilder.Invoke(Object instance, IApplicationBuilder builder)
   at Microsoft.AspNetCore.Hosting.ConfigureBuilder.<>c__DisplayClass4_0.<Build>b__0(IApplicationBuilder builder)
   at Microsoft.AspNetCore.Hosting.ConventionBasedStartup.Configure(IApplicationBuilder app)
```

I tried downgrading bolero   FSharp.SystemTextJson by adding
nuget FSharp.SystemTextJson < 0.18.24
to the paket.dependencies file and that didn't work either.  So I reverted all my packages to the previous versions (rolled back my paket.dependencies file and my packet.lock file) and it worked again.  My dependencies file is:
```
source https://api.nuget.org/v3/index.json
framework: net6.0
storage: none


nuget FSharp.Core content: none
nuget Bolero.Build 
nuget Bolero.HotReload.Server 
nuget Microsoft.AspNetCore.Components.WebAssembly.Server >= 6.0.0
nuget Microsoft.AspNetCore.Components.WebAssembly.DevServer >= 6.0.0
nuget Npgsql
nuget Evolve
nuget Dapper.FSharp
```

my paket.lock file is:
```
STORAGE: NONE
RESTRICTION: == net6.0
NUGET
  remote: https://api.nuget.org/v3/index.json
    Bolero (0.20.17)
      Elmish (>= 3.1 < 4.0)
      FSharp.Core (>= 6.0)
      FSharp.SystemTextJson (>= 0.17.4)
      Microsoft.AspNetCore.Components.WebAssembly (>= 6.0)
      Microsoft.Extensions.Http (>= 6.0)
    Bolero.Build (0.20.17)
    Bolero.HotReload (0.20.4)
      Bolero (>= 0.20.11)
      FSharp.Core (>= 6.0)
      HtmlAgilityPack (>= 1.11.39)
      Microsoft.AspNetCore.SignalR.Client (>= 6.0)
    Bolero.HotReload.Server (0.20.4)
      Bolero.HotReload (0.20.4)
      Bolero.Server (>= 0.20.11)
      FSharp.Core (>= 6.0)
    Bolero.Server (0.20.17)
      Bolero (0.20.17)
      FSharp.Core (>= 6.0)
      FSharp.SystemTextJson (>= 0.17.4)
    Dapper (2.0.123)
    Dapper.FSharp (3.2)
      Dapper (>= 2.0.123)
      FSharp.Core (>= 6.0.1)
    Elmish (3.1)
      FSharp.Core (>= 4.6.2)
    Evolve (3.0)
    FSharp.Core (6.0.4) - content: none
    FSharp.SystemTextJson (0.17.4)
      FSharp.Core (>= 4.7)
      System.Text.Json (>= 4.6)
    HtmlAgilityPack (1.11.43)
    Microsoft.AspNetCore.Authorization (6.0.5)
      Microsoft.AspNetCore.Metadata (>= 6.0.5)
      Microsoft.Extensions.Logging.Abstractions (>= 6.0.1)
      Microsoft.Extensions.Options (>= 6.0)
    Microsoft.AspNetCore.Components (6.0.5)
      Microsoft.AspNetCore.Authorization (>= 6.0.5)
      Microsoft.AspNetCore.Components.Analyzers (>= 6.0.5)
    Microsoft.AspNetCore.Components.Analyzers (6.0.5)
    Microsoft.AspNetCore.Components.Forms (6.0.5)
      Microsoft.AspNetCore.Components (>= 6.0.5)
    Microsoft.AspNetCore.Components.Web (6.0.5)
      Microsoft.AspNetCore.Components (>= 6.0.5)
      Microsoft.AspNetCore.Components.Forms (>= 6.0.5)
      Microsoft.Extensions.DependencyInjection (>= 6.0)
      Microsoft.JSInterop (>= 6.0.5)
      System.IO.Pipelines (>= 6.0.3)
    Microsoft.AspNetCore.Components.WebAssembly (6.0.5)
      Microsoft.AspNetCore.Components.Web (>= 6.0.5)
      Microsoft.Extensions.Configuration.Binder (>= 6.0)
      Microsoft.Extensions.Configuration.Json (>= 6.0)
      Microsoft.Extensions.Logging (>= 6.0)
      Microsoft.JSInterop.WebAssembly (>= 6.0.5)
    Microsoft.AspNetCore.Components.WebAssembly.DevServer (6.0.5)
    Microsoft.AspNetCore.Components.WebAssembly.Server (6.0.5)
    Microsoft.AspNetCore.Connections.Abstractions (6.0.5)
      Microsoft.Extensions.Features (>= 6.0.5)
      System.IO.Pipelines (>= 6.0.3)
    Microsoft.AspNetCore.Http.Connections.Client (6.0.5)
      Microsoft.AspNetCore.Http.Connections.Common (>= 6.0.5)
      Microsoft.Extensions.Logging.Abstractions (>= 6.0.1)
      Microsoft.Extensions.Options (>= 6.0)
    Microsoft.AspNetCore.Http.Connections.Common (6.0.5)
      Microsoft.AspNetCore.Connections.Abstractions (>= 6.0.5)
    Microsoft.AspNetCore.Metadata (6.0.5)
    Microsoft.AspNetCore.SignalR.Client (6.0.5)
      Microsoft.AspNetCore.Http.Connections.Client (>= 6.0.5)
      Microsoft.AspNetCore.SignalR.Client.Core (>= 6.0.5)
    Microsoft.AspNetCore.SignalR.Client.Core (6.0.5)
      Microsoft.AspNetCore.SignalR.Common (>= 6.0.5)
      Microsoft.AspNetCore.SignalR.Protocols.Json (>= 6.0.5)
      Microsoft.Extensions.DependencyInjection (>= 6.0)
      Microsoft.Extensions.Logging (>= 6.0)
      System.Threading.Channels (>= 6.0)
    Microsoft.AspNetCore.SignalR.Common (6.0.5)
      Microsoft.AspNetCore.Connections.Abstractions (>= 6.0.5)
      Microsoft.Extensions.Options (>= 6.0)
    Microsoft.AspNetCore.SignalR.Protocols.Json (6.0.5)
      Microsoft.AspNetCore.SignalR.Common (>= 6.0.5)
    Microsoft.Extensions.Configuration (6.0.1)
      Microsoft.Extensions.Configuration.Abstractions (>= 6.0)
      Microsoft.Extensions.Primitives (>= 6.0)
    Microsoft.Extensions.Configuration.Abstractions (6.0)
      Microsoft.Extensions.Primitives (>= 6.0)
    Microsoft.Extensions.Configuration.Binder (6.0)
      Microsoft.Extensions.Configuration.Abstractions (>= 6.0)
    Microsoft.Extensions.Configuration.FileExtensions (6.0)
      Microsoft.Extensions.Configuration (>= 6.0)
      Microsoft.Extensions.Configuration.Abstractions (>= 6.0)
      Microsoft.Extensions.FileProviders.Abstractions (>= 6.0)
      Microsoft.Extensions.FileProviders.Physical (>= 6.0)
      Microsoft.Extensions.Primitives (>= 6.0)
    Microsoft.Extensions.Configuration.Json (6.0)
      Microsoft.Extensions.Configuration (>= 6.0)
      Microsoft.Extensions.Configuration.Abstractions (>= 6.0)
      Microsoft.Extensions.Configuration.FileExtensions (>= 6.0)
      Microsoft.Extensions.FileProviders.Abstractions (>= 6.0)
      System.Text.Json (>= 6.0)
    Microsoft.Extensions.DependencyInjection (6.0)
      Microsoft.Extensions.DependencyInjection.Abstractions (>= 6.0)
      System.Runtime.CompilerServices.Unsafe (>= 6.0)
    Microsoft.Extensions.DependencyInjection.Abstractions (6.0)
    Microsoft.Extensions.Features (6.0.5)
    Microsoft.Extensions.FileProviders.Abstractions (6.0)
      Microsoft.Extensions.Primitives (>= 6.0)
    Microsoft.Extensions.FileProviders.Physical (6.0)
      Microsoft.Extensions.FileProviders.Abstractions (>= 6.0)
      Microsoft.Extensions.FileSystemGlobbing (>= 6.0)
      Microsoft.Extensions.Primitives (>= 6.0)
    Microsoft.Extensions.FileSystemGlobbing (6.0)
    Microsoft.Extensions.Http (6.0)
      Microsoft.Extensions.DependencyInjection.Abstractions (>= 6.0)
      Microsoft.Extensions.Logging (>= 6.0)
      Microsoft.Extensions.Logging.Abstractions (>= 6.0)
      Microsoft.Extensions.Options (>= 6.0)
    Microsoft.Extensions.Logging (6.0)
      Microsoft.Extensions.DependencyInjection (>= 6.0)
      Microsoft.Extensions.DependencyInjection.Abstractions (>= 6.0)
      Microsoft.Extensions.Logging.Abstractions (>= 6.0)
      Microsoft.Extensions.Options (>= 6.0)
      System.Diagnostics.DiagnosticSource (>= 6.0)
    Microsoft.Extensions.Logging.Abstractions (6.0.1)
    Microsoft.Extensions.Options (6.0)
      Microsoft.Extensions.DependencyInjection.Abstractions (>= 6.0)
      Microsoft.Extensions.Primitives (>= 6.0)
    Microsoft.Extensions.Primitives (6.0)
      System.Runtime.CompilerServices.Unsafe (>= 6.0)
    Microsoft.JSInterop (6.0.5)
    Microsoft.JSInterop.WebAssembly (6.0.5)
      Microsoft.JSInterop (>= 6.0.5)
    Npgsql (6.0.4)
      System.Runtime.CompilerServices.Unsafe (>= 6.0)
    System.Diagnostics.DiagnosticSource (6.0)
      System.Runtime.CompilerServices.Unsafe (>= 6.0)
    System.IO.Pipelines (6.0.3)
    System.Runtime.CompilerServices.Unsafe (6.0)
    System.Text.Encodings.Web (6.0)
      System.Runtime.CompilerServices.Unsafe (>= 6.0)
    System.Text.Json (6.0.4)
      System.Runtime.CompilerServices.Unsafe (>= 6.0)
      System.Text.Encodings.Web (>= 6.0)
    System.Threading.Channels (6.0)
```

Maybe the issue has to do with a change with FSharp.Core 6.0.5?

**Comment by kgday** [comment]
Thankyou

**Comment by kgday** [comment]
But I still get the same error with FSharp.SystemTextJson 0.19.13.  Also if I do your suggested workarround of locking to less then 0.18 (paket finds 0.17.4) I get the error also after I upgraded. I downgraded System.Text.Json from 6.05 to 6.04 also without joy. So it seems to have something to do with the newer versons of the microsoft libraries.

**Comment by kgday** [comment]
System.MissingMethodException: Method not found: 'Void System.Text.Json.Serialization.JsonFSharpConverter..ctor(System.Text.Json.Serialization.JsonUnionEncoding, System.String, System.String, System.Text.Json.JsonNamingPolicy, Boolean, Boolean, Boolean, System.Collections.Generic.IDictionary`2<System.Type,System.Text.Json.Serialization.JsonFSharpOptions>)'.
   at Bolero.Remoting.Server.RemotingService..ctor(PathString basePath, Type ty, Object handler, FSharpOption`1 configureSerialization)
   at <StartupCode$Bolero-Server>.$Remoting.AddRemoting@202-9.Invoke(IServiceProvider services)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitFactory(FactoryCallSite factoryCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitRootCache(ServiceCallSite callSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteVisitor`2.VisitCallSite(ServiceCallSite callSite, TArgument argument)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitIEnumerable(IEnumerableCallSite enumerableCallSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.VisitRootCache(ServiceCallSite callSite, RuntimeResolverContext context)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteVisitor`2.VisitCallSite(ServiceCallSite callSite, TArgument argument)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteRuntimeResolver.Resolve(ServiceCallSite callSite, ServiceProviderEngineScope scope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.CreateServiceAccessor(Type serviceType)
   at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.GetService(Type serviceType, ServiceProviderEngineScope serviceProviderEngineScope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.GetService(Type serviceType)
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetRequiredService(IServiceProvider provider, Type serviceType)
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetRequiredService[T](IServiceProvider provider)
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetServices[T](IServiceProvider provider)
   at Bolero.Remoting.Server.ServerRemotingExtensions.UseRemoting(IApplicationBuilder this)
   at WGD.Server.Startup.Configure(IApplicationBuilder app, IWebHostEnvironment env, DbMigrator migrator) in C:\Development\Projects\Upwork\WGD\src\WGD.Server\Startup.fs:line 68
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Span`1& arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at Microsoft.AspNetCore.Hosting.ConfigureBuilder.Invoke(Object instance, IApplicationBuilder builder)
   at Microsoft.AspNetCore.Hosting.ConfigureBuilder.<>c__DisplayClass4_0.<Build>b__0(IApplicationBuilder builder)
   at Microsoft.AspNetCore.Hosting.ConventionBasedStartup.Configure(IApplicationBuilder app)
   at Microsoft.AspNetCore.Mvc.Filters.MiddlewareFilterBuilderStartupFilter.<>c__DisplayClass0_0.<Configure>g__MiddlewareFilterBuilder|0(IApplicationBuilder builder)
   at Microsoft.WebTools.BrowserLink.Net.HostingStartup.<>c__DisplayClass1_0.<Configure>b__0(IApplicationBuilder app)
   at Microsoft.AspNetCore.Watch.BrowserRefresh.HostingStartup.<>c__DisplayClass1_0.<Configure>b__0(IApplicationBuilder app)
   at Microsoft.AspNetCore.HostFilteringStartupFilter.<>c__DisplayClass0_0.<Configure>b__0(IApplicationBuilder app)
   at Microsoft.AspNetCore.Hosting.WebHost.BuildApplication()
The thread 0xfa4 has exited with code 0 (0x0).
Exception thrown: 'System.MissingMethodException' in System.Private.CoreLib.dll
An unhandled exception of type 'System.MissingMethodException' occurred in System.Private.CoreLib.dll
Method not found: 'Void System.Text.Json.Serialization.JsonFSharpConverter..ctor(System.Text.Json.Serialization.JsonUnionEncoding, System.String, System.String, System.Text.Json.JsonNamingPolicy, Boolean, Boolean, Boolean, System.Collections.Generic.IDictionary`2<System.Type,System.Text.Json.Serialization.JsonFSharpOptions>)'.

**Comment by kgday** [comment]
> Also if I do your suggested workarround of locking to less then 0.18 (paket finds 0.17.4) I get the error also after I upgraded. I downgraded System.Text.Json from 6.05 to 6.04 and that worked.

Actually downgrading just the FSharp.SystemTextJson < 0.18 did work.  I just had to delete all the bin and obj folders in the project before doing a paket install again.

**Comment by Tarmil** [comment]
This is fixed in v0.20.18.
